dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/ImagePart.java

```diff
@@ -15,27 +15,34 @@
  *
  * @author steff
  */
-public class ImagePart implements EntityPart{
-    private Texture texture;
+public class ImagePart implements EntityPart {
+
+    private FileHandle fileHandle;
+    private int width;
+    private int hight;
+    private Texture textur = null;
```

Comment:
Private Texture Textur is missing an e for spelling (texture)

Reply:
fix